tests/test_clientsecrets.py

```diff
@@ -32,7 +33,8 @@
 DATA_DIR = os.path.join(os.path.dirname(__file__), 'data')
 VALID_FILE = os.path.join(DATA_DIR, 'client_secrets.json')
 INVALID_FILE = os.path.join(DATA_DIR, 'unfilled_client_secrets.json')
-NONEXISTENT_FILE = os.path.join(__file__, '..', 'afilethatisntthere.json')
+NONEXISTENT_FILE = os.path.join(os.path.dirname(__file__),
```

Comment:
I've expressed it poorly, but yes, it is a serious guiding principle for the textual layout and flow of software, and I'm going to continue expressing it poorly, but it goes something like prefer
```
my_really_long_named_field = my_really_long_function(
    a + b, 1 + 2 + 3 + 4 + 5 + 6 + 7 + 8 + 9 + 10, c * d)
```
to
```
my_really_long_named_field = my_really_long_function(a + b, 1 + 2 + 3 + 4 + 5 + 6 + 7 + 8 +
                                                     9 + 10, c * d)
```
because the former is more easily understood. My guess is that it's rooted somewhere in the fact that in the latter form the subexpression and the superexpression both span the line wrap but in the former form only the superexpression is across the line break. Of course a comma-separated series of expressions constituting the parameters of a function call aren't an expression in an of themselves, but "place function call parameters together on the same line if possible rather than spreading them across lines" is a natural enough extension of the general rule.

Far from the most important thing in software authoring, of course, but perfectly serious and not a joke.